src/tunedev/EGP.go

```diff
@@ -0,0 +1,7 @@
+package tunedev
+
+type EgyptianPound float32 // Egyptian Pound
+
+func (e EgyptianPound) toNaira() Naira {
+	return Naira(e * 22.3171)
+}
```

Comment:
All the code in separate files should be in a single file.

Reply:
Yeh, i was trying to extend the feature in such a way that those other currency types have there own conversion to other currencies. For example lets say you want to convert Egyptian pound to Benin republic cefa, so the files would house those implementation logic.

Reply:
Yeah, I see, but the files are much and are not `required` to make your desired logic come to life.

Reply:
Well i intend extending the other currency types to convert to other currencies, currently catching up on missed lectures. this is like a base for that extension